src/displayapp/screens/PineTimeStyle.cpp

```diff
@@ -108,7 +108,7 @@ PineTimeStyle::PineTimeStyle(DisplayApp* app,
 
   notificationIcon = lv_label_create(lv_scr_act(), nullptr);
   lv_obj_set_style_local_text_color(notificationIcon, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(0x000000));
-  lv_obj_align(notificationIcon, sidebar, LV_ALIGN_IN_TOP_MID, 0, 40);
+  lv_obj_align(notificationIcon, sidebar, LV_ALIGN_IN_TOP_MID, 0, 51);
```

Comment:
Wouldn't it make more sense to make the positions 0, 25 and 50? Even if it wouldn't be pixel perfect, it would be independent of the font. In #718 this actually gets set to 50 as well. Also there will be a gap when the bluetooth icon isn't shown, so maybe it would be better to automatically realign the icons like in the digital watchface.

Reply:
I just wanted to fix a small alignment bug by changing as little as possible. Furthermore I assumed the non-realigning icons are a design choice (so I didn't want to touch it)

I don't get the argument with different fonts, we got the fonts fixed here. It's not like you upload different fonts to the pinetime?

The PR #718 looks very promising. Much better interaction to get to the color settings

Reply:
I meant to say it's basically a magic number that depends on the exact font and characters we are using, instead of a constant distance of 25.

Reply:
Yes, totally a magic number and font dependent :sweat_smile: 

Using the 0, 25, 50 proposal the following distances of the icons are seen:
- 6 px - battery - 9px bluetooth - 6px - notification - 9px - FRI
- 2 px - charger - 6px bluetooth - 6px - notification - 9px - FRI

![battery 0 25 50](https://user-images.githubusercontent.com/9076163/144669245-fcaa3c53-fb75-4765-ad32-c7112c104416.png) ![charger 0 25 50](https://user-images.githubusercontent.com/9076163/144669249-882ba117-87eb-4be3-96d0-3a03bbb59667.png)

If we can get consensus, that the 0, 25, 50 strategy is preferable to the magic 2, 25, 51 strategy I'll gladly change the PR. Everything is better than the overlapping icons ;)

Reply:
Oh actually I didn't realize that it started from 2 instead of 0. Let's just leave it like it is now then and leave the rest of the tweaking to if/when the icons are made dynamically aligning.